src/app/views/app-sections/AppTitle.tsx

```diff
@@ -49,16 +62,33 @@ export function appTitleDisplayOnMobileScreen(
         onClick={() => toggleSidebar()}
       />
       <div style={{ padding: 10 }} role={'heading'} aria-level={1}>
-        {displayGraphLabel(classes)}
+        {displayGraphLabel(classes, permissionType)}
+      </div>
+      <div style={{ marginTop: 15 }}>
+        {
+          !minimised &&
+          <>
+            {permissionsModeButton(changeMode, permissionType)}
+          </>
+        }
       </div>
     </>
   </Stack>;
 }
 
-function displayGraphLabel(classes: any): React.ReactNode {
+function displayGraphLabel(classes: any, permissionType: boolean): React.ReactNode {
   return (
     <Label className={classes.graphExplorerLabel}>
-      Graph Explorer
+      Graph Explorer {permissionType ? "(as user)" : "(as Teams app)"}
     </Label>
   )
 }
+
+function permissionsModeButton(changeMode: Function, permissionType: boolean) {
+  return (
+    <IconButton
+      iconProps={{ iconName: 'Cat' }}
+      onClick={() => changeMode(!permissionType)}
+      ariaLabel="button to switch between user and Teams app" />
+  )
```

Comment:
This button is for testing purposes, and is not the final form.

Reply:
![image](https://user-images.githubusercontent.com/47487758/123350451-e5b4f700-d50f-11eb-961f-e2c76ccc9d2a.png)

Reply:
omg I laughed out loud at this 🤣